website_sale_one_step_checkout/models/res_partner.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+# Odoo, an open source suite of business apps
+# This module copyright (C) 2015 bloopark systems (<http://bloopark.de>).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import fields, models
+
+
+class ResPartner(models.Model):
+
+    """Add Fields to res.partner."""
+
+    _inherit = 'res.partner'
+
+    street_name = fields.Char(string='Street name')
+    street_number = fields.Char(tring='Street number')
```

Comment:
i think you should remove this two fields and just deal with the normal "street" field(s).
As the format "street" + "number" is typical german and not a international format